### PR TITLE
Improved layout

### PR DESCRIPTION
**Pull request by andriy1507**
Improved layout as it should be.

### CODE REVIEW

File: app/src/main/java/com/goryachok/forecastapp/view/activity/MainActivity.kt [hunk 1]
```diff
@@ -57,38 +56,39 @@ class MainActivity : AppCompatActivity() {
...
         setContentView(R.layout.activity_main)
         supportActionBar?.title = ""
         initViewPager()
+        viewModel.locationProvider.setTask {
+            (pagerAdapter.getItem(forecast_viewPager.currentItem) as? MyFragment)?.onLocationRequest(
+                it
+            )
+            viewModel.locationCache = it
+            viewModel.requestCache = ""
+            viewModel.locationProvider.stop()
```
Comment (EugeneKryvoshey) [inline app/src/main/java/com/goryachok/forecastapp/view/activity/MainActivity.kt]: Please, use 'with' scope function instead

File: app/src/main/java/com/goryachok/forecastapp/view/activity/MainActivity.kt [hunk 2]
```diff
@@ -57,38 +56,39 @@ class MainActivity : AppCompatActivity() {
...
+        }
     }
 
     override fun onStart() {
         super.onStart()
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
-            connectivityListener.start()
+            viewModel.connectivityListener.start()
+            viewModel.connectionStatus.observe(this, Observer {
```
Comment (EugeneKryvoshey) [inline app/src/main/java/com/goryachok/forecastapp/view/activity/MainActivity.kt]: Please, use 'with' scope function instead

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -5,15 +5,17 @@
...
     <androidx.viewpager.widget.ViewPager
         android:id="@+id/forecast_viewPager"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
+        android:layout_marginBottom="@dimen/snackbar_padding"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="parent" />
+        app:layout_constraintTop_toBottomOf="parent"/>
```
Comment (EugeneKryvoshey) [inline app/src/main/res/layout/activity_main.xml]: Please, remove redundant constraints.

File: app/src/main/res/layout/current_weather_fragment.xml [hunk 4]
```diff
@@ -9,119 +9,127 @@
...
         android:id="@+id/curWindDir_textView"
+        style="@style/WeatherText.SmallParameters"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
         app:layout_constraintEnd_toStartOf="@+id/curHumid_textView"
         app:layout_constraintHorizontal_bias="0.5"
         app:layout_constraintStart_toEndOf="@+id/curWindSpeed_textView"
         app:layout_constraintTop_toTopOf="@+id/curWindSpeed_textView"
+        android:gravity="center"
         tools:layout_editor_absoluteY="608dp"
```
Comment (EugeneKryvoshey) [inline app/src/main/res/layout/current_weather_fragment.xml]: Please, remove this line.

File: app/src/main/res/layout/current_weather_fragment.xml [hunk 5]
```diff
@@ -9,119 +9,127 @@
...
         android:id="@+id/curHumid_textView"
+        style="@style/WeatherText.SmallParameters"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
         app:layout_constraintEnd_toStartOf="@+id/curPress_textView"
         app:layout_constraintHorizontal_bias="0.5"
         app:layout_constraintStart_toEndOf="@+id/curWindDir_textView"
         app:layout_constraintTop_toTopOf="@+id/curWindSpeed_textView"
+        android:gravity="center"
         tools:layout_editor_absoluteY="608dp"
```
Comment (EugeneKryvoshey) [inline app/src/main/res/layout/current_weather_fragment.xml]: Please, remove this

**Review by EugeneKryvoshey — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/goryachok/forecastapp/view/activity/MainActivity.kt (hunk 1 above)
Comment (andriy1507) [reply app/src/main/java/com/goryachok/forecastapp/view/activity/MainActivity.kt]: Done

File: app/src/main/java/com/goryachok/forecastapp/view/activity/MainActivity.kt (hunk 2 above)
Comment (andriy1507) [reply app/src/main/java/com/goryachok/forecastapp/view/activity/MainActivity.kt]: Done

File: app/src/main/res/layout/activity_main.xml (hunk 3 above)
Comment (andriy1507) [reply app/src/main/res/layout/activity_main.xml]: Done

File: app/src/main/res/layout/current_weather_fragment.xml (hunk 4 above)
Comment (andriy1507) [reply app/src/main/res/layout/current_weather_fragment.xml]: Done

File: app/src/main/res/layout/current_weather_fragment.xml (hunk 5 above)
Comment (andriy1507) [reply app/src/main/res/layout/current_weather_fragment.xml]: Done

**Review by EugeneKryvoshey — APPROVED** [review]

**Pull request merged**